src/components/charts/DataTable.vue

```diff
@@ -79,4 +79,7 @@ export default {
 </script>
 
 <style scoped>
+  .invert-text {
+    color: white;
```

Comment:
Any color should be a style guide color, even if it's white

Reply:
good one